apis/v1beta1/validation/httproute.go

```diff
@@ -40,6 +41,9 @@ var (
 
 	// All valid path characters per RFC-3986
 	validPathCharacters = "^(?:[A-Za-z0-9\\/\\-._~!$&'()*+,;=:@]|[%][0-9a-fA-F]{2})+$"
+
+	// Minimum value of timeout in milliseconds
```

Comment:
```suggestion
	// Minimum value of timeout in microseconds
```

Reply:
why microseconds @frankbu ?

Reply:
Golang Duration values are in microseconds. Notice that the value `of int(time.Millisecond)` is `1000000`

Reply:
Oh, wait, it's nanoseconds, actually.
```suggestion
	// Minimum value of timeout in nanoseconds
```

Reply:
I suppose you could avoid the detail by just saying:
```suggestion
	// Minimum value of timeout fields
```